src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+body{
   margin: 0;
+
+  --color-text: #00ACDC;
+
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+}
+
+.header{
+  max-width: 100%;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.link-image{
+  height: 40px;
+  width: 40px;
+}
+
+
+.nav-list{
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav-item{
+  margin-right: 20px;
+}
+
+.nav-item:nth-last-child(1){
+margin-right: 0;
+}
+
+.nav-link{
+  height: 60px;
+  color:#000;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)